projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -37,53 +35,50 @@
+- Add site migration endpoint. [#36253]
+- Compile typescript extensions in Jetpack. [#35904]
+- Contact Form: add deprecation warnings to Contact Form module codebase. [#36040]
+- Contact Form: refactor editor to use forms package. [#36089]
+- Contact Form: refactor endpoints to use forms package. [#36087]
 - Editor: Update to the most recent version of the @automattic/calypso-color-schemes package. [#36227]
-- General: update to the most recent version of @automattic/calypso-color-schemes [#36187]
+- General: update to the most recent version of @automattic/calypso-color-schemes. [#36187]
+- Login block: Link to subscription management page with site URL as search query.
+- Member login block: link to individual subscription management page. [#36200]
 - Newsletter settings: update section title to "subscriptions" [#36343]
-- Paywall: Use Subscriber Login block for the login link [#36308]
+- Paywall: Use Subscriber Login block for the login link. [#36308]
+- Pass API parameter to indicate when deleting a memberships product should cancel subscriptions. [#35735]
 - Pinterest Block: deprecate in favor of the Core Pinterest embed block. [#36075]
-- Refactor user_is_paid_subscriber to compare against tiers [#35587]
+- Refactor user_is_paid_subscriber to compare against tiers. [#35587]
 - Removed the feature flag for newsletter categories settings. [#36073]
 - RNMobile: Remove code associated with Story block for the mobile native version. [#36151]
-- Show Browse sites when wp-admin interface is selected and using nav unification [#36198]
-- Subscriber Login: Prevent the HTTP 301 redirection for Atomic sites [#36311]
-- Subscriber Login: Remove the premium content cookie specified for the blog on logging out [#35441]
-- Subscriptions: fix empty email for pending confirm paywall on .com [#36321]
+- Show Browse sites when wp-admin interface is selected and using nav unification. [#36198]
+- Subscriber Login: Prevent the HTTP 301 redirection for Atomic sites. [#36311]
+- Subscriber Login: Remove the premium content cookie specified for the blog on logging out. [#35441]
+- Subscriptions: fix empty email for pending confirm paywall on .com. [#36321]
 - Subscriptions: Track newsletter category creation. [#36326]
 - Subscriptions: update the settings screen URL. [#36323]
-- Subscription Site: Polishing Newsletter settings [#36218]
-- Subscription Site: Polishing the Subscribe block toggle [#36177]
-- Subscription Site: Update Subscribe block after the post nudge [#36107]
-- Update code references in docs and comments [#36234]
-- Updated package dependencies. [#36309]
-- Updated package dependencies. [#36325]
-- Update lockfiles [#36195]
+- Subscription Site: Polishing Newsletter settings. [#36218]
+- Subscription Site: Polishing the Subscribe block toggle. [#36177]
+- Subscription Site: Update Subscribe block after the post nudge. [#36107]
+- Uncheck users-new.php Invite user on load if WooCommerce plugin is activated. [#36140]
```

Comment:
?

Reply:
Yeah, I'll have to re-word that and put it in its place alphabetically.